principles.html

```diff
@@ -201,16 +201,22 @@ <h4 class="f_panelHead">Accessibility requirements related to text alternatives
     <ul class="listwithp">
       <li><a href="http://www.w3.org/WAI/WCAG20/quickref/#text-equiv" shape="rect">Guideline 1.1 - Text Alternatives</a></li>
     </ul>
-    <p class="listintro"><strong><acronym>UAAG</acronym> 1.0</strong></p>
-    <ul class="listwithp">
-      <li><a href="http://www.w3.org/TR/UAAG10/guidelines.html#gl-content-access" shape="rect">Guideline 2 - Ensure user access to all content</a></li>
-    </ul>
-    <p class="listintro"><strong><acronym>ATAG</acronym> 1.0</strong></p>
-    <ul class="listwithp">
-      <li><a href="http://www.w3.org/TR/ATAG10/#gl-access-support" shape="rect">Guideline 1 - Support accessible authoring practices</a></li>
-      <li><a href="http://www.w3.org/TR/ATAG10/#gl-prewritten-descs" shape="rect">Guideline 3 - Support the creation of accessible content</a></li>
-      <li><a href="http://www.w3.org/TR/ATAG10/#gl-make-accessible" shape="rect">Guideline 7 - Ensure that the authoring tool is accessible to authors with disabilities</a></li>
-    </ul>
+    <p class="listintro"><strong><acronym>UAAG</acronym> 2.0</strong></p>
+<ul class="listwithp">
+  <li>orginal: <a href="http://www.w3.org/TR/UAAG10/guidelines.html#gl-content-access" shape="rect">Guideline 2 - Ensure user access to all content</a></li>
+  <li>replaced 2 with: <a href="https://www.w3.org/TR/UAAG20/#principle-perceivable" shape="rect">Principle 1 - Ensure that the user interface and rendered content are perceivable</a></li>
```

Comment:
Please try to stay on the level of "Guidelines" unless nothing corresponds. The "Principles" level seems too broad. This section is about text alternatives, so the closest corresponding UAAG 2.0 guideline is "Guideline 1.1 - Provide access to alternative content".